deploy/runtime/python/install_python.ps1

```diff
@@ -0,0 +1,143 @@
+ param (
+    [Parameter(Mandatory=$true)][string]$version,
+    [Parameter(Mandatory=$true)][string]$config
+ )
+
+$PYTHON_CONFIG_FILE = "python_config.json"
+
+function Invoke-CommandExitOnError {
+    param([string]$command)
+    Write-Debug "Run command [$command]"
+
+    $command_args = -split $command
+    $exec = $command_args[0]
+    if ($command_args.Count -gt 1) {
+        $args = $command_args[1..($command_args.Count - 1)]
+    } else {
+        $args = @()
+    }
+
+    $result = Start-Process $exec -ArgumentList $args -Wait -NoNewWindow -PassThru -RedirectStandardOutput nul
+    if ($result.ExitCode -ne 0) {
+        $msg = "failed to run [$command]: ExitCode={0}" -f $result.ExitCode
+        Write-Error $msg
+        exit $result.ExitCode
+    }
+}
+
+
+function Invoke-WebRequestExitOnError {
+    param([string]$url, [string]$filename)
+    Write-Debug "Invoke-WebRequest $url"
+
+    try {
+        Invoke-WebRequest -OutFile $filename $url
+    }
+    catch {
+        throw "failed to download $url"
+    }
+}
+
+
+function Get-Python {
```

Comment:
This function has two behavioral inconsistencies with the Linux/Mac install script.
1. It does not support the FORCE_DOWNLOAD env variable to allow skipping the download for devs. This is minor, but we should make it consistent.
2. The script will fail if the to-be-extracted Python directory already exists.  The Miniconda install command now includes `-u' to update whereas it used to fail. I forgot to make a comparable change here.  Just need to call `Remove-Item` on the directory if it exists.

Reply:
Yes, good points. Would you please add those options?